rd_stats.h

```diff
@@ -547,6 +547,15 @@ struct stats_filesystem {
 
 #define STATS_FILESYSTEM_SIZE	(sizeof(struct stats_filesystem))
 
+/* Structure for HBA statistics */
+struct stats_hba {
+	unsigned long long f_txframes		__attribute__ ((aligned (16)));
+	unsigned long long f_rxframes		__attribute__ ((aligned (16)));
```

Comment:
Defined as unsigned long long here.
Read as unsigned int in rd_stats.c.
Please be consistent (use the same type as used by the kernel. If you don't feel browsing the kernel sources, then use unsigned long long).